cmd/notary-server/dev-config.json

```diff
@@ -7,7 +7,8 @@
 	"trust_service": {
 		"type": "local",
 		"hostname": "",
-		"port": ""
+		"port": "",
+		"key_algorithm": "ed25519"
```

Comment:
Note that's in dev-config.json which is set up to run a notary server locally, fully in memory, outside a container.